ftdc/sys/net.go

```diff
@@ -57,13 +57,13 @@ func (netStatser *netStatser) Stats() any {
 		}
 	}
 
-	if netTCPSummary, err := netStatser.fs.NetTCPSummary(); err == nil {
+	if netTCPSummary, err := netStatser.fs.NetTCPSummary(); err == nil && netTCPSummary != nil {
```

Comment:
Can you check the source code to see if it's possible for NetTCPSummary() to return a nil value and* a nil error? I'm not keen on accepting this change -- I prefer to be more explicit with a nolint if the linter is actually wrong. To avoid confusing the reader.